OpenRA.Game/World.cs

```diff
@@ -66,11 +66,10 @@ public void EndGame()
 			}
 		}
 
-		public bool ObserveAfterWinOrLose;
 		Player renderPlayer;
 		public Player RenderPlayer
 		{
-			get { return renderPlayer == null || (ObserveAfterWinOrLose && renderPlayer.WinState != WinState.Undefined) ? null : renderPlayer; }
+			get { return renderPlayer == null || renderPlayer.Spectating ? null : renderPlayer; }
```

Comment:
Ah, sorry, this needs to be `|| renderPlayer.WinState != WinState.Undefined ? null : renderPlayer` after all, otherwise the 'All players' view will break. Please amend the first commit I made with the change.